test/modules/DeploymentTester/Settings.cs

```diff
@@ -66,7 +66,7 @@ public class Settings
             this.DeploymentUpdatePeriod = deploymentUpdatePeriod;
         }
 
-        public static Settings Current => DefaultSettings.Value;
+        internal static Settings Current => DefaultSettings.Value;
 
         public string DeviceId { get; }
```

Comment:
I think `DeviceId` can also be `internal` right? Since it's in the same namespace

Reply:
all properties below can be internal; since class is internal already; it is not necessary to change these readonly properties to internal.